poi/src/test/java/org/apache/poi/ss/format/TestCellFormat.java

```diff
@@ -1046,4 +1046,11 @@ void testNamedColors() {
                 .map(CellFormatPart.NAMED_COLORS::get)
                 .forEach(Assertions::assertNotNull);
     }
+
+    @Test
+    void testElapsedSecondsRound() {
```

Comment:
* this looks like a better way to test this
* the tests where we read xlsx files and match their results can be easily broken and the PRs are hard to review because you can't see the diff in the xlsx files